.github/workflows/app.yml

```
@@ -71,6 +71,7 @@ jobs:
         name: 'SETUP: copy the JDL file for the backend and app'
       - run: $SCRIPT_DIR/generate-react-native-app.sh
         name: 'GENERATING: generate react-native app'
+      - run: npx expo-doctor@latest
```

```suggestion
        name: 'GENERATING: generate react-native app'
```
Remove expo-doctor since it seems to cause CI to fail.